components/scripts/gradle/02-validate-local-build-caching-same-location.sh

```diff
@@ -246,7 +265,10 @@ Now that the second build has finished successfully, you are ready to measure in
 Gradle Enterprise how well your build leverages Gradle’s local build caching
 functionality for the invoked set of Gradle tasks.
 
-${USER_ACTION_COLOR}Press <Enter> to measure the build results.${RESTORE}
+Some of the build scan data will be fetched from the build scans produced by the two builds
+to assist you in your investigation.
+
+${USER_ACTION_COLOR}Press <Enter> to fetch build scan data and measure the build results.${RESTORE}
```

Comment:
I think we missed adding the same text to Gr exp 4 and Maven exp 3 in the past. If you agree, can you please add it there, too?

Reply:
I omitted this text from Gradle Exp 04 and Maven Exp 03 because the build scan data has already been fetched by this point. Maybe we add it but edit it to be past-tense (rather than describing something that is about to happen)?